src/NoteWorkspace.ts

```diff
@@ -116,6 +119,7 @@ export class NoteWorkspace {
       ) as boolean,
       defaultFileExtension: c.get('defaultFileExtension') as string,
       noteCompletionConvention: c.get('noteCompletionConvention') as NoteCompletionConvention,
+      uniqueIdTemplate: c.get('uniqueIdTemplate') as string,
```

Comment:
Maybe add a getter like below is well:
![image](https://github.com/kortina/vscode-markdown-notes/assets/14327513/9758fe1a-2ed9-4c31-87d5-72ded5a2f555)